### PR TITLE
New package: vscode-oss-1.26.1

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

**Comment by shizonic** [comment]
This only fails because the github api limit is exceeded. A workaround would be to set an API key but for this I need one generated by a repo maintainer.

**Comment by shizonic** [comment]
Just added a fix for this bug:
https://github.com/Microsoft/vscode/issues/23946

File: srcpkgs/vscode-oss/REMOVE [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```
Comment (maxice8) [inline srcpkgs/vscode-oss/REMOVE]: no need for shebang

File: srcpkgs/vscode-oss/REMOVE [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+if [ "$UPDATE" = "no" ]; then
+    case "$ACTION" in
+        post)
+            if [ -d /usr/lib/code-oss ]; then
```
Comment (maxice8) [inline srcpkgs/vscode-oss/REMOVE]: paths should be relative, no first /

File: srcpkgs/vscode-oss/REMOVE [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+                rm -rf /usr/lib/code-oss
```
Comment (maxice8) [inline srcpkgs/vscode-oss/REMOVE]: ditto

**Comment by shizonic** [comment]
Thanks @maxice8 for reviewing. Will we fixed asap.

**Pull request closed without merging**